File: backend/components/metiers/threed/tests_centralises/legacy/samples/policy/test_policy_sample.py
"""
Test ultra avancé pour policy_sample.py
"""

from threed.legacy.samples.policy.policy_sample import policy_sample


def test_policy_admin():
    assert policy_sample("admin") == "full-access"


def test_policy_user():
    assert policy_sample("user") == "restricted"


# ...autres tests avancés, edge cases, sécurité...

File: backend/components/metiers/threed/tests_centralises/legacy/samples/policy/test_policy_sample.js
// Test ultra avancé pour policy_sample.js
const policySample = require('../../../legacy/samples/policy/policy_sample');

describe('Policy Sample', () => {
  it('doit donner full-access à admin', () => {
    expect(policySample('admin')).toBe('full-access');
  });
  it('doit donner restricted à un user', () => {
    expect(policySample('user')).toBe('restricted');
  });
  // ...autres tests avancés, edge cases, sécurité...
});
